messages/src/lib.rs

```diff
@@ -143,6 +161,11 @@ impl PortMask {
             Ok(Self(1 << index))
         }
     }
+
+    /// Return the number of transceivers addressed by `self.
+    pub const fn n_transceivers(&self) -> usize {
```

Comment:
Nit: is this name standard / pulled from somewhere else?

Otherwise, I'd vote for something obvious, even if it's a bit longer – maybe `selected_transceiver_count`?

Reply:
It's totally arbitrary! I'll rename to your suggestion, much clearer thank you.